src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -2,56 +2,68 @@
+| **Key name**                  | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                                                                                           |
+| ----------------------------- | --------------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Manufacturer                  | RECOMMENDED           | [string][]    | Manufacturer of the equipment that produced the composite instances. Corresponds to DICOM Tag 0008, 0070 `Manufacturer`                                                                                                                                                   |
+| ManufacturersModelName        | RECOMMENDED           | [string][]    | Manufacturer's model name of the equipment that produced the composite instances. Corresponds to DICOM Tag 0008, 1090 `Manufacturers Model Name`                                                                                                                          |
+| DeviceSerialNumber            | RECOMMENDED           | [string][]    | The serial number of the equipment that produced the composite instances. Corresponds to DICOM Tag 0018, 1000 `DeviceSerialNumber`. A pseudonym can also be used to prevent the equipment from being identifiable, so long as each pseudonym is unique within the dataset |
+| StationName                   | RECOMMENDED           |               | Institution defined name of the machine that produced the composite instances. Corresponds to DICOM Tag 0008, 1010 `Station Name`                                                                                                                                         |
```

Comment:
what datatype is this?

Reply:
Every instance I can quickly find in OpenNeuro is a string, which matches my intuition.